src/platyPS/platyPS.psm1

```diff
@@ -203,10 +203,20 @@ function New-MarkdownHelp
                     }
                     else
                     {
+                        
                         $a = @{
                             Name = $commandName
                         }
 
+                        # fixes https://github.com/PowerShell/platyPS/issues/389 for generating docs using relative script paths
+                        # check if external command and update command to use the path to the external script.
+                        # By defualt Get-Command does not search the current location for scripts
+                        $isLocalScript = $commandName -match "\\.*.\.ps1\b"
```

Comment:
I don't understand why do you want such a complicated regex. What's wrong with a simple `.EndsWith('.ps1')` ?

This regex doesn't allow `foo.ps1`, is there a reason? I miss it.